app/src/main/res/values-ru/strings.xml

```diff
@@ -1,6 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources xmlns:tools="http://schemas.android.com/tools" tools:ignore="MissingTranslation">
 
+    <!-- Do NOT translate these -->
+    <string name="download_progress" translatable="false">%1$,d / %2$,d кб</string>
```

Comment:
Changes required

Reply:
This line needs translation.  Can you remove not translatable?

Reply:
> This line needs translation. Can you remove not translatable?

Sorry, it's my fault. It's fixed